samples/cloudwatch/README.md

```diff
@@ -92,25 +122,59 @@ kubectl delete pod -n amazon-cloudwatch -l app=cwagent-prometheus
+widgets in CloudWatch to visualize the collected metric data. Metrics
+collected by the CloudWatch agent for Prometheus are stored in the
 `/aws/containerinsights/[cluster-name]/prometheus` log group.
 
-For example, these are the steps to create a graph of `go_memstats_alloc_bytes`
-(memory usage) of SAS Viya microservices that are written in Go:
+To access the metrics:
 
 1. From the AWS web console, navigate to CloudWatch.
 2. In the navigation bar on the left, choose `Metrics`.
 3. Select the `ContainerInsights/Prometheus` namespace.
-4. Select `ClusterName, job, namespace, pod, sas_service_base`.
-5. Locate the `go_memstats_alloc_bytes` metric.
-6. Click on the arrow next to the metric and select `Search for this only`.
-7. Select the checkbox for the SAS Viya services that you want to include in the graph.
+
+The Cloudwatch **Metrics** view displays a set of tiles, with each tile corresponding to a common set of dimensions, You can hover over a tile's dimensions to display the complete list of dimensions associated with the tile. 
+
+The dimensions for a metric correspond to the metric's Prometheus labels, and are used to specify attributes about the source of the metric. Metrics use different dimensions, depending on the type of data being collected, For example, a metric collecting the memory of a CAS node might include dimensions such as the cluster name, the CAS node name, and the CAS node type. A metric collecting the memory usage of a SAS service might also use the cluster name, but include the SAS service base and SAS service name. 
+
+All of the metrics that use a common set of dimensions are grouped under a tile that is labeled with the list of dimensions. In order to find the a specific metric, you must find the tile with the corresponding set of dimensions.
+
+Because each metric has many dimensions and because many SAS Viya metrics are collected for Cloudwatch, it can
+be difficult to find a specific metric or metrics. You can use
+the reference tables in [CloudWatch SAS Viya Metrics](reference.md) to locate
+SAS Viya metrics and identify the tile with which they are associated. The tables provide these cross-reference listings: 
+
+- the metrics associated with each set of dimensions
+- the dimensions and metrics associated with each type of source (such as CAS, SAS services written in Go, or SAS services written in Java)
+- the dimensions associated with each each metric
+
+After you find the metric you are interested in, you can create a graph from
+the metric data. For example, these are the steps to create a graph of
+`go_memstats_alloc_bytes` (memory usage) of SAS Viya services that are
+written in Go:
+
+1. Go to [CloudWatch SAS Viya Metrics](reference.md) and expand the
+**By Metric** table.
+2. Locate the `go_memstats_alloc_bytes` metric. The table indicates that it is
+associated with the `container,job,namespace,node,pod,sas_service_base,service`
```

Comment:
It's under 'ClusterName,job,namespace,node,pod,sas_service_base,service'

Reply:
fixed